examples/tactile_pose_estimation.py

```diff
@@ -281,18 +275,11 @@ def run_learning_loop(cfg):
 
     outer_optim = optim.Adam(learnable_params, lr=learning_rate)
     batch_size = cfg.train.batch_size
-    num_batches = cfg.train.num_batches
-    measurements = [
-        (
-            dataset["img_feats"][0:time_steps].unsqueeze(0).repeat(batch_size, 1, 1),
```

Comment:
@psodhi Maybe you had already explained this and I forgot, but I'm curious about why this repeats the same trajectory data for each batch element and batch. Is this a placeholder? 

If I wanted to change this to select different trajectory data, what changes should I make?

Reply:
Yes, it is a placeholder. It is currently repeating a single trajectory. To select different trajectories, you would have to pass in different episodes indexes (`cfg.episode`) in `dataset = theg.TactilePushingDataset(dataset_path, sdf_path, cfg.episode, device).` And concatenate the outputs to get a batch dataset. Note that different `cfg.episode` trajectories can have different lengths, so it may require some min length trimming / padding.